src/index.html

```diff
@@ -7,6 +7,102 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar">
+        <div class="calendar__day">
+          <div class="day-of-week day-of-week--1 day-of-week--mon"></div>
+        </div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```

Reply:
calendar__day is enough

use :nth-child of `calendar` to add text in `calendar__day` cell